src/v2/view-builder/views/shared/polling.js

```diff
@@ -6,16 +6,16 @@ export default {
     this.pollingInterval = newRefreshInterval || this.options.currentViewState.refresh;
```

Comment:
lets create two variables here.. 
`this.fixedPollingInterval = this.options.currentViewState.refresh;`
`this.dynamicPollingInterval = newRefreshInterval`

Reply:
created